parser/src/main/java/main/ParserMain.java

```diff
@@ -53,7 +53,7 @@ public static void main(String[] args) {
         log.info("Parser finish his work.");
 
         Cache cache = ctx.getBean(Cache.class);
-        cache.initializeCache(bookRepo);
+        cache.initializeCache(bookRepo, args[0]);
```

Comment:
I know (from you) that is not needed any more, but keep in mind that kind of getting arguments will be first place where you receive an exception